src/praisonai-bot/praisonai_bot/bots/_streaming.py

```
@@ -230,6 +241,23 @@ async def start(self) -> str:
+        return (
+            self._config.mode == StreamingMode.PROGRESS
+            and self._config.progress_style == "feed"
+        )
+
+    def _fold_progress_event(self, event: "StreamEvent") -> None:
+        """Fold a stream event into the structured progress feed (feed style)."""
+        from praisonaiagents.streaming.progress import merge_progress_line
+
+        try:
+            self._progress_lines = merge_progress_line(self._progress_lines, event)
+        except Exception as e:  # never break streaming on a compositor error
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)